src/dynamodb.js

```diff
@@ -66,14 +66,11 @@ async function destroyTables (dynamoClient, uniqueIdentifier) {
   }
 }
 
-async function getConnection () {
-  if (process.env.DYNAMODB_ENDPOINT) {
-    return buildConnectionAndConfig({ url: process.env.DYNAMODB_ENDPOINT });
-  }
-
-  const docker = new Docker();
-  const environment = new Environment();
-
+/**
+ * @param params.inMemory Whether to run local DynamoDB in in-memory mode.
+ * 'false' persists data to disk.
+ */
+async function launchDynamoContainer ({ inMemory } = { inMemory: false }, docker = new Docker()) {
```

Comment:
Nit: can the docker option be moved into the first argument hash?

Reply:
Done.